src/test/java/com/centrilli/stepDefinitions/Contacts_StepDefinitions.java

```diff
@@ -0,0 +1,184 @@
+package com.centrilli.stepDefinitions;
+
+import com.centrilli.pages.ContactsPage;
+import com.centrilli.pages.LoginPage;
+import com.centrilli.utilities.BrowserUtil;
+import com.centrilli.utilities.Driver;
+import com.sun.jna.Library;
+import com.zaxxer.sparsebits.SparseBitSet;
```

Comment:
Unused import ( you create these imports when you miss type something when working on your code) - and when you fix that miss type the import statement still stays here. Usually these dont give problems but sometimes do. Please remove this import statement. TY